### PR TITLE
Add init command

### LINKED ISSUE #39: Automate copying seed config example file.
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
Currently it's necessary to copy the sample config file from a specific GOPATH location.

**What you expected to happen**:
Provide a reposeed command to simplify that.
`reposeed init .` <- initialize seed config file in "." directory with sample data.

The sample data should be integrated into the binary via packr and copied into seed-config.yml on the command `reposeed init .`

**Anything else we need to know?**:
Rough idea, so happy to get a bit more input.
This would remove the need for knowledge about GOPATH and any manual copying and also makes releasing binaries possible without any manual copy steps.

### PR DESCRIPTION
**Pull request by erbesharat**
**What this PR does / why we need it**:
Add init command to automate copying seed config example file

**Which issue this PR fixes**:
fixes #39 

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by stp-ip** [comment]
This still needs a bit of documentation. Either via overwriting the help page of the flag lib or in the readme.

File: README.md [hunk 1]
```diff
@@ -32,7 +34,9 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 ----
 
 *Code is licensed under the [Apache License, Version 2.0](/LICENSE).*  
+
 *Documentation/examples are licensed under [Creative Commons BY-SA 4.0](/docs/LICENSE).*  
+
```
Comment (stp-ip) [inline README.md]: These whitespace lines are too much. Now it's a new line and an empty line.

Comment (erbesharat) [reply README.md]: I just added a \n at the end of each license line. So I think it was working correctly before. Will remove those \n.

Comment (stp-ip) [reply README.md]: Yeah I saw that, but you already regenerated the README so this "bug" should have been showing up before. Not sure, why it came up now.

Comment (erbesharat) [reply README.md]: It's kinda weird. because even without \n it works fine and each license is on a separate line.

Comment (stp-ip) [reply README.md]: Weird, but as long as the actual bug is fixed \o/

File: README.md [hunk 2]
```diff
@@ -31,8 +33,8 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 
 ----
 
-*Code is licensed under the [Apache License, Version 2.0](/LICENSE).*  
-*Documentation/examples are licensed under [Creative Commons BY-SA 4.0](/docs/LICENSE).*  
+*Code is licensed under the [Apache License, Version 2.0](/LICENSE).*
```
Comment (stp-ip) [inline README.md]: The 2 empty spaces are necessary for normal markdown. Maybe that was the issue on Github itself.

**Pull request merged**